app/views/posts/show.html.slim

```diff
@@ -1,17 +1,41 @@
+= stylesheet_link_tag 'comments'
+
 .dflex.justify-content-center
   .card style="padding:2% 5% 2% 5%"
-    small.text-muted
-      | Posted by&#160
-      i
+    span
+      small.text-muted
+        | Posted by
         = User.find(@post.user_id).username
-      | &#160&#183
+      - if user_signed_in? && current_user == @post.user
+        div
+          = link_to "Edit", edit_community_post_path(@post.id), method: :get, class: "btn btn-primary btn-sm"
+          = link_to "Delete" , community_post_path(@post.id), data: { confirm: "Are you sure?" }, method: :delete, class: "btn btn-danger btn-sm"
     h4.card-title
       = @post.title
     hr
     p.card-text
       = @post.content
     hr
-    span
-      - if user_signed_in? && current_user.id == @post.user_id
-        = link_to "Edit", edit_community_post_path(@post.id), method: :get, class: "btn btn-primary"
-        = link_to "Delete" , community_post_path(@post.id), data: { confirm: "Are you sure?" }, method: :delete, class: "btn btn-danger"
+  p.comments-header
+    | Comments(
```

Comment:
ir iekš slim supports interpolācijai: `| Comments (#{@comments.size})`


BTW - iegooglē kkur postu par ActiveRecord size vs count vs length un izlasi. Iemācīsies varbūt kkādus trikus.

Reply:
Beigas pamainiju uz `count` , since lidz tam bridim nebija memory nekas ieloadots un counts takes care of that.